controllers/ebpf/agent_controller.go

```diff
@@ -71,6 +69,13 @@ const (
 	bpfNetNSMountPath  = "/var/run/netns"
 )
 
+const (
+	EnvDedupeJustMark     = "DEDUPER_JUST_MARK"
+	EnvDedupeMerge        = "DEDUPER_MERGE"
+	DedupeJustMarkDefault = "true"
+	DedupeMergeDefault    = "false"
+)
+
```

Comment:
those const are used only locally no need for CAPs

Reply:
I'm actually using these in `consoleplugin_objects.go` so I need to expose them.

Would you prefer to move these to `constants.go` ?

Reply:
oh sorry my mistake that should be fine then
/lgtm